stock_measuring_device/models/product_packaging.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+import logging
+
+from odoo import _, fields, models
```

Comment:
```suggestion
from odoo import fields, models
```

same -> `self.env._`